strings/ork_replacement.json

```
@@ -0,0 +1,87 @@
+{
+	"ork": {
+		"gun": "blasta",
+		"ammo": "dakka",
+		"bullets": "dakka",
+		"rockets": "bigger dakka",
+		"grenade": "bomb",
+		"rifle": "shoota",
+		"pistol": "short boomstick",
+		"shotgun": "boomstick",
+		"launcher": "bomb throwa",
+		"sword": "harder choppa",
+		"knife": "choppa",
+		"spear": "long choppa",
+		"disk": "disky",
+		"nuke disk": "disky",
+		"captain": "warboss",
+		"security" : "warboyz",
+		"cap ": "boss ",
+		" cap": " boss",
+		"science": "smarty thinking",
+		"scientist" :"brain boy",
+		"engineer": "mekkanik",
+		"engineering": "tinkering",
+		"technician": "mekboy",
+		"botanist": "green feeder",
+		"botany": "green feed",
+		"felinid": "talking kitty",
+		"robotics": "kanna maka",
+		"roboticist": "kanna boy"
+		"cat": "kitty",
+		"moth": "bug",
+		"mothperson": "bugperson",
```

```suggestion
		"mothperson": "bugman",
```
person has too many syllables, "bugboy" or just "bug" is also acceptable

---

or of course "bugga" but that might give people the wrong idea